homeassistant/components/broadlink/climate.py

```diff
@@ -0,0 +1,85 @@
+"""Support for Broadlink climate devices."""
+from typing import Any
+
+from homeassistant.components.climate import (
+    ATTR_TEMPERATURE,
+    ClimateEntity,
+    ClimateEntityFeature,
+    HVACAction,
+    HVACMode,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import PRECISION_HALVES, Platform, UnitOfTemperature
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .const import DOMAIN, DOMAINS_AND_TYPES
+from .device import BroadlinkDevice
+from .entity import BroadlinkEntity
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    config_entry: ConfigEntry,
+    async_add_entities: AddEntitiesCallback,
+) -> None:
+    """Set up the Broadlink climate entities."""
+    device = hass.data[DOMAIN].devices[config_entry.entry_id]
+
+    if device.api.type in DOMAINS_AND_TYPES[Platform.CLIMATE]:
+        async_add_entities([BroadlinkThermostat(device)])
+
+
+class BroadlinkThermostat(ClimateEntity, BroadlinkEntity):
```

Comment:
Please invert the inheritance order.

Reply:
https://github.com/home-assistant/core/pull/110927